mesos-swarm-marathon/configure-mesos-cluster.sh

```diff
@@ -11,6 +11,7 @@
 ###########################################################
 
 echo "starting mesos cluster configuration"
```

Comment:
There is an explicit abort if the network doesn't come up.  I am holding off on aborting on the remaining errors, as I want to see where and how the failures manifest.